panels/info/cc-info-panel.c

```diff
@@ -585,6 +582,7 @@ get_primary_disc_info (CcInfoPanel *self)
   g_list_free (points);
   g_hash_table_destroy (hash);
 
+  self->priv->cancellable = g_cancellable_new ();
```

Comment:
Why not doing this in `cc_info_panel_init ()`?

Reply:
Avoids allocating it until the disk info is actually queried. Currently, that’s inevitable, but I guess it could change in future. I don’t want our copy of this patch to diverge from upstream, though, since it’s a simple cherry pick from upstream.

Reply:
Fair enough. I did realize it was from upstream, just curious on the rationale.